src/components/banner.jsx

```diff
@@ -9,7 +9,7 @@ import LanguageSelector from './language-selector';
 function Banner({ children, slug }) {
   const { t } = useTranslation();
   return (
-    <div className="usa-section--dark">
+    <section className="usa-section--dark">
```

Comment:
Why change to section?

Reply:
For consistency; you use sections in the other components.